tests/testthat/test-helper-functions.R

```diff
@@ -144,3 +144,26 @@ test_that("`get_file_info()` and `file.info()` outputs are equivalent", {
   expect_true(is.na(na_info[["ctime"]]))
   expect_true(is.na(na_info[["atime"]]))
 })
+
+test_that("find_newer_files_than() works", {
+  old_file <- tempfile()
+  new_file1 <- tempfile()
+  new_file2 <- tempfile()
+  brio::write_file("", old_file)
+  Sys.sleep(1)
```

Comment:
One second is a long time. Does .01 work? What's the lowest resolution for mtime?

Reply:
> What's the lowest resolution for mtime?

I couldn't find the answer, but it seems `.01` works.